db/fixtures/miq_product_features.yml

```diff
@@ -154,6 +154,10 @@
       :feature_type: view
       :hidden: true
       :identifier: miq_request_reload
+    - :name: Logs
+      :description: Display Request Logs
+      :feature_type: view
+      :identifier: miq_request_show_logs
```

Comment:
how does this work if you have `miq_request_view` but no child features?  For example:  https://github.com/ManageIQ/manageiq/blob/5912c45c703a3639652bb3cce0f69c24df5a6dc2/db/fixtures/miq_user_roles.yml#L1214

I assume it shows the logs.

What does the UI do if you modif your role and select all the features below `miq_request_view`, such as `miq_request_show_list`, `miq_request_show`, etc. but exclude `miq_request_show_logs`?  I hope that disables it in the UI for your role.

Reply:
Discussed this with @DavidResende0 and he verified that the default behavior for user roles like plain [user](https://github.com/ManageIQ/manageiq/blob/5912c45c703a3639652bb3cce0f69c24df5a6dc2/db/fixtures/miq_user_roles.yml#L769-L854) which have `miq_request_view` but no child nodes is to be shown in the UI with all child nodes selected and therefore they'd default to showing request logs.

Users can configure roles and remove the option for the request logs for each user role as they see fit. 

This is basically what I was asking... if it's opt-in or opt-out.  It should be opt-out for most if not all user roles.